### PR TITLE
LPS-130805 Update integration test of Instance Resource

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java [hunk 1]
```diff
@@ -393,25 +416,25 @@ public ProcessMetric addProcessMetric(long companyId, String version)
 	}
 
 	public void addSLAInstanceResult(
-			long companyId, Instance instance, boolean onTime)
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java:396]: Hey @MarinhoFeliphe, rename to `addSLAInstanceResults` instead. Thx

**Comment by rafaprax** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java [hunk 2]
```diff
@@ -804,16 +828,69 @@ private Document _creatWorkflowMetricsSLAInstanceResultDocument(
...
 			"uid",
 			_digest(
 				"WorkflowMetricsSLAInstanceResult", companyId, instanceId,
-				processId, slaDefinitionId)
+				processId, slaDefinitionId));
+
+		return documentBuilder.build();
+	}
+
+	private Document _creatWorkflowMetricsSLAInstanceResultDocument(
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java:853]: Hey @MarinhoFeliphe, Can we adapt the code to have only one `_creatWorkflowMetricsSLAInstanceResultDocument`?

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java [hunk 3]
```diff
@@ -147,18 +147,26 @@ public Instance getProcessInstance(Long processId, Long instanceId)
 		overdueFilterAggregation.addChildAggregation(
 			_resourceHelper.createOverdueScriptedMetricAggregation());
 
-		TermsAggregation slaDefinitionIdTermsAggregation = _aggregations.terms(
-			"slaDefinitionId", "slaDefinitionId");
+		BooleanQuery slaInstanceResultBooleanQuery = _queries.booleanQuery();
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java:150]: Hey @MarinhoFeliphe, please, inline the `slaInstanceResultBooleanQuery.addMustQueryClauses(...)` into the `_aggregations.filter("slaInstanceResult", ...)` one.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Nice job @MarinhoFeliphe. Congrats

**Comment by MarinhoFeliphe** [comment]
ci:help

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow-metrics

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**